cli/command/format.go

```diff
@@ -73,6 +79,7 @@ func NewFormatCommand(curveadm *cli.CurveAdm) *cobra.Command {
 	flags := cmd.Flags()
 	flags.StringVarP(&options.filename, "formatting", "f", "format.yaml", "Specify the configure file for formatting chunkfile pool")
 	flags.BoolVar(&options.showStatus, "status", false, "Show formatting status")
+	flags.BoolVar(&options.stopFormat, "stop", false, "stop formatting status")
```

Comment:
The usage message of `stop` option should be `Stop formatting progress`.

BTW: i think we shoud use subcommands to instead of all options, like:

```
curveadm format start
curveadm format status
curveadm format stop
```

Reply:
resolve

current subcommands not support in format?

Reply:
so I keep the same write as status

Reply:
> 

I real forget why we use option instead of subcommands, but you can change it if you interested it.

Reply:
> The usage message of `stop` option should be `Stop formatting progress`.
> 
> BTW: i think we shoud use subcommands to instead of all options, like:
> 
> ```
> curveadm format start
> curveadm format status
> curveadm format stop
> ```

Usage message:

```
flags.BoolVar(&options.stopFormat, "stop", false, "Stop formatting progress")
```

not

```
flags.BoolVar(&options.stopFormat, "stop", false, "stop formatting status")
```

Reply:
resove the usage

i think we should keep the option with pre version ,stop and status 

subcommands can rewrite in next version for status , stop and others

Reply:
> 

agree!